Source/HelixToolkit.Wpf.SharpDX/Themes/Generic.xaml

```diff
@@ -54,7 +54,7 @@
                                 </hx:CameraController.InputBindings>
                                 -->
                                 <AdornerDecorator x:Name="PART_AdornerLayer" DataContext="{TemplateBinding DataContext}" HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
-                                    <hx:DPFCanvas x:Name="PART_Canvas"/>
+                                    <hx:DPFCanvas x:Name="PART_Canvas" EffectsManager="{Binding EffectsManager}" RenderTechniquesManager="{Binding RenderTechniquesManager}"/>

```

Comment:
The bound values for these new `DependencyProperty` are `IEffectsManager` and `IRenderTechniquesManager` respectively. This allows third-parties to supply their own implementation of both. I was surprised and happy to learn that this data binding still works even when the underlying property specifies an interface.